rosbag2_cpp/src/rosbag2_cpp/cache/message_cache.cpp

```diff
@@ -87,7 +88,8 @@ void MessageCache::wait_for_buffer()
   std::swap(primary_buffer_, secondary_buffer_);
 }
 
-std::shared_ptr<MessageCacheBuffer> MessageCache::consumer_buffer()
+std::shared_ptr<rosbag2_cpp::cache_interfaces::BaseCacheBufferInterface> MessageCache::
+consumer_buffer()
```

Comment:
Perhaps it could be shortened as:
```suggestion
std::shared_ptr<CacheBufferInterface> MessageCache::consumer_buffer()
```